### PR TITLE
Changing layout for for palace

### PR DESCRIPTION
**Pull request by Huy-Ngo**
This PR renders the sidebar fixed and shows relbar (prev/next), which make it easier for users to navigate back and forth.

### CODE REVIEW

**Comment by Huy-Ngo** [comment]
Since I was writing this on Windows, on which palace is not installed, the reference was yoten. I'll recompile this on my Linux VM after finish writing tutorial.

**Comment by McSinyx** [comment]
When it's ready for review please checkout the generated files to `gh-pages`, since it's quite noisy for proper review.  Alternatively you can have a commit for source change and one purely as output of `make html` and I can just review the first commit and ignore the later.

**Comment by McSinyx** [comment]
Too bad closing and reopening the PR doesn't reset AppVeyor.  Try pushing something to see if the check disappears.

File: src/conf.py [hunk 1]
```diff
@@ -59,6 +59,8 @@
 # so a file named "default.css" will overwrite the builtin "default.css".
 html_static_path = ['_static']
```
Comment (McSinyx) [inline src/conf.py]: delet dis

**Review by McSinyx — APPROVED** [review]

**Comment by McSinyx** [comment]
The new layout looks really nice!

**Pull request merged**